UI/src/main/webapp/resources/js/Builder.js

```diff
@@ -0,0 +1,214 @@
+class Builder {
+    _nameUser
+    _ad
+
+    constructor(nameUser, ad = null) {
+        this._nameUser = nameUser;
+        this._ad = ad;
+    }
+
+
+    createHead() {
+        let userMenu = document.createElement("div");
+        userMenu.className = "header__user-menu";
+        userMenu.appendChild(document.createElement("div")).className = "header__button-add";
+        userMenu.lastElementChild.appendChild(document.createElement("button")).className = "button-add";
+        userMenu.lastElementChild.lastElementChild.textContent = "��������";
+
+        userMenu.appendChild(document.createElement("div")).className = "header__name";
+        userMenu.lastElementChild.textContent = this._nameUser;
+
+        userMenu.appendChild(document.createElement("button")).className = "header__button-log-out";
+        userMenu.lastElementChild.textContent = "�����";
+        return userMenu;
+    }
+
+    _createStar() {
+        let ratingStars = document.createElement("div");
+        ratingStars.className = "post-info__rating-stars";
+        for (let i = 0; i < 5; i++) {
+            let input = ratingStars.appendChild(document.createElement("input"));
+            input.type = "radio";
+            input.id = "post-" + this._ad.id + "-star-" + (5 - i);
+            input.disabled = true;
+            if (this._ad.rating !== undefined && 5 - i === this._ad.rating) {
+                input.checked = true;
+            }
+            ratingStars.appendChild(document.createElement("label")).htmlFor = input.id;
+        }
+        return ratingStars;
+    }
+
+    _createHead() {
```

Comment:
You have two methods (here and at line 11) with the same name, how can someone understand purpose of each of them?